operators/awx-operator/2.7.2/manifests/awx-operator.v2.7.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1357 @@
+          - get
+          - list
+          - create
+          - delete
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - get
+          - list
+          - create
+          - delete
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - networking.k8s.io
+          resources:
+          - ingresses
+          verbs:
+          - get
+          - list
+          - create
+          - delete
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - awx-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - apps
+          resources:
+          - deployments/scale
+          - statefulsets/scale
+          verbs:
+          - patch
+        - apiGroups:
+          - ""
+          resources:
+          - pods/exec
+          - pods/attach
+          - pods/log
+          verbs:
+          - create
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - awx.ansible.com
+          resources:
+          - '*'
+          - awxbackups
+          - awxrestores
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: awx-operator-controller-manager
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: true
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - automation
+  - ansible
+  links:
+  - name: AWX Operator
+    url: https://github.com/ansible/awx-operator
+  maintainers:
+  - email: awx-project@googlegroups.com
+    name: AWX Team
+  maturity: alpha
+  minKubeVersion: 1.22.15
+  provider:
+    name: Ansible
+    url: github.com/ansible/awx-operator
+  version: 2.7.2
+  replaces: awx-operator.v
```

Comment:
@TheRealHaoLiu this should be `awx-operator.v2.7.1` instead

So the test failed when adding the bundle to the catalog because of the replaces:

```
Error: Invalid bundle awx-operator.v2.7.2, replaces nonexistent bundle awx-operator.v
```